tutorials/commands/flux-jobs.rst

```diff
@@ -0,0 +1,344 @@
+       f2SLdmS4F achu     false       F      1      1   0.057s catalyst160
+       f2SLfFRLb achu     false       F      1      1   0.056s catalyst160
+       f2RYByw1h achu     true       CD      1      1   0.560s catalyst160
+       f2RYAVwjM achu     true       CD      1      1   0.453s catalyst160
+
+------------------
+Advanced Filtering
+------------------
+
+In this particular example, it's not too annoying to run ``flux jobs -a`` because we only have
+12 jobs total.  However, over time, you may have hundreds, if not thousands, of jobs to list.  It can
+become difficult to filter and find your jobs.
+
+There are many ways to filter the job listing to limit the output to the jobs you are interested in.
+Here are some of the most common options.
+
+The simplest way to limit job output is to specify the jobid of the jobs you wish to list.  This is typically
+done because you want to monitor the status of some finite number of your jobs.
+
+.. code-block:: console
+
+    $ flux jobs f2RYAVwjM f2RYByw1h f2SLfFRLb f2SLdmS4F
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+       f2RYAVwjM achu     true       CD      1      1   0.453s catalyst160
+       f2RYByw1h achu     true       CD      1      1   0.560s catalyst160
+       f2SLfFRLb achu     false       F      1      1   0.056s catalyst160
+       f2SLdmS4F achu     false       F      1      1   0.057s catalyst160
+
+Here we list the jobids of the ``/bin/true`` and ``/bin/false`` jobs to get the results of just those jobids.
+
+By default ``flux jobs`` will limit output to 1000 jobs.  If the number of jobs is getting too large (or you want to show even more jobs) you can adjust this via the ``--count`` option.
+
+.. code-block:: console
+
+    $ flux jobs --count=4
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+       f2TRD37vP achu     sleep       S      1      -        - 
+       f2TREX7Cj achu     sleep       S      1      -        - 
+       f2TRG16V5 achu     sleep       S      1      -        - 
+       f2TRBZ8e3 achu     sleep       R      1      1   47.88m catalyst160
+
+    $ flux jobs --count=4 --filter=inactive
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+       f2TRHV5mR achu     sleep      CA      1      -        - 
+       f2SLdmS4F achu     false       F      1      1   0.057s catalyst160
+       f2SLfFRLb achu     false       F      1      1   0.056s catalyst160
+       f2RYByw1h achu     true       CD      1      1   0.560s catalyst160
+
+Here we pass ``--count=4`` to limit the of jobs output from ``flux jobs`` default output and when we
+specify that we should only list inactive jobs.
+
+We already saw above that ``--filter`` can be used to filter jobs on "pending", "running" or "inactive"
+state.  But we can also filter on the result of a job.  In the following example, we show that you can
+list "completed", "failed", or "canceled" jobs respectively.
```

Comment:
I could be wrong, but I think that commas go inside the sets of quotation marks when you are listing multiple items, so it would be `"pending," "running," or "inactive" state.`.

Reply:
the first hit on google says yes, they go inside the quotes!

Reply:
Eh in this case I think it's clearer with the comma outside the quoted item, since what's inside the quotes is literal input to the command.

Reply:
Good call @garlick; probably much clearer to just leave them outside the quotation marks. 👍